packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -17,6 +17,8 @@ import type {InteractiveGraphState} from "./types";
 import type {GraphRange} from "@khanacademy/perseus-core";
 import type {UserEvent} from "@testing-library/user-event";
 
+const baseMafsProps = getBaseMafsGraphPropsForTests();
```

Comment:
While I was in this file, I took this as an opportunity to reduce the number of function calls being made.